lib/backup/encryptor/open_ssl.rb

```diff
@@ -56,7 +56,7 @@ def encrypt_with
       # Always sets a password option, if even no password is given,
       # but will prefer the password_file option if both are given.
       def options
-        opts = ["aes-256-cbc"]
+        opts = ["aes-256-cbc -md sha512 -pbkdf2 -iter 1000"]
```

Comment:
Why did you select these particular options?

Reply:
Well, don't remember exactly, maybe I just tried some configuration which worked without deprecated keys.